build-config/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -26,36 +26,41 @@
       <property name="ignoreFinal" value="false"/>
       <property name="tokens" value="ANNOTATION_FIELD_DEF,PARAMETER_DEF,VARIABLE_DEF,METHOD_DEF"/>
     </module>
+    <module name="AnnotationLocation">
+      <property name="allowSamelineMultipleAnnotations" value="false"/>
+      <property name="allowSamelineSingleParameterlessAnnotation" value="false"/>
+      <property name="allowSamelineParameterizedAnnotation" value="false"/>
+    </module>
     <module name="ArrayTypeStyle">
-      <property name="severity" value="error"/>
       <message key="array.type.style" value="Array must be defined as Foo[] array, not Foo array[]"/>
     </module>
-    <module name="AvoidStarImport">
-      <property name="severity" value="error"/>
+    <module name="AvoidStarImport"/>
+    <module name="CatchParameterName">
+      <property name="format" value="^(ex|e)$"/>
     </module>
-    <module name="ConstantName">
+      <module name="ConstantName">
       <property name="format" value="^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$|^log([A-Z][a-zA-Z0-9]*)?$|^[a-z][a-zA-Z0-9]*Log$"/>
     </module>
     <module name="CovariantEquals"/>
-    <module name="DefaultComesLast">
-      <property name="severity" value="error"/>
-    </module>
+    <module name="DefaultComesLast"/>
     <module name="EmptyBlock">
       <property name="option" value="text"/>
       <property name="tokens" value="LITERAL_DO,LITERAL_ELSE,LITERAL_FINALLY,LITERAL_IF,LITERAL_FOR,LITERAL_TRY,LITERAL_WHILE,STATIC_INIT"/>
     </module>
     <module name="EqualsHashCode"/>
-    <module name="ExplicitInitialization">
-      <property name="severity" value="error"/>
-    </module>
-    <module name="FallThrough">
-      <property name="severity" value="error"/>
-    </module>
+    <module name="ExplicitInitialization"/>
+    <module name="FallThrough"/>
     <module name="GenericWhitespace"/>
     <module name="IllegalImport"/><!-- other imports disallowed by regex below -->
     <module name="IllegalInstantiation">
       <property name="classes" value="Boolean"/>
-      <property name="severity" value="error"/>
+    </module>
+    <module name="ImportOrder">
```

Comment:
Just to note that Google changed to a [single group of fully alphabetical](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements). Not sure we want the churn, but it is interesting.